src/components/TrackFilePicker.demo.js

```diff
@@ -0,0 +1,30 @@
+import Demo, {props as P} from 'react-demo'
+// See https://github.com/rpominov/react-demo for how to make a demo
+
+import React from "react";
+
+import TrackFilePicker from "./TrackFilePicker";
+
+export default (<Demo
+  props={{
+    tracks: P.json([{"files": [{"name": "fileA1.vg", "type": "graph"},
+                              {"name": "fileA2.gbwt", "type": "haplotype"}]},
+                    {"files": [{"name": "fileB1.gbwt", "type": "haplotype"},
+                              {"name": "fileB2.gam", "type": "read"}]},
+                    {"files": [{"name": "fileC1.xg", "type": "graph"}]}]),
```

Comment:
Multi-file tracks like this are good to test, but they might be a bit counterintuitive, since despite the "TrackFilePicker" name I think the right behavior here would be to allow selecting any whole track object with a file of a matching type.